optimade/server/main.py

```diff
@@ -35,24 +21,12 @@
         "by developing a common REST API."
     ),
     version="0.10.0",
+    docs_url="/extensions/docs",
+    redoc_url="/extensions/redoc",
```

Comment:
I forgot how nice these interfaces are :) but perhaps they should be at `/optimade/extensions/docs`?

Reply:
They aren't? I thought they were...

Reply:
Indeed they are not. I cannot find a good way to reimplement it several times, so they will _only_ live under `/optimade/extensions/*`.